ql/src/test/queries/clientpositive/dataconnector_mysql_bit_datatype.q

```diff
@@ -0,0 +1,24 @@
+--!qt:database:mysql:q_test_mysql_datatype_mapping.sql
+-- CREATE with comment
+CREATE CONNECTOR mysql_data_type_dc
+TYPE 'mysql'
+URL 'jdbc:mysql://localhost:3306/qtestDB'
+COMMENT 'test connector'
+WITH DCPROPERTIES (
+"hive.sql.dbcp.username"="root",
+"hive.sql.dbcp.password"="qtestpassword");
+
+CREATE REMOTE DATABASE mysql_data_type_db USING mysql_data_type_dc with DBPROPERTIES("connector.remoteDbName"="qtestDB");
+
+USE mysql_data_type_db;
+SHOW TABLES;
+
+SHOW CREATE TABLE work_calendar;
+-- SELECT * FROM work_calendar;  //Wait for HIVE-26192 to be fixed
```

Comment:
Please uncomment the query and add "explain select * ... ", adding a predicate on the bit column would be good.

Reply:
Yes, i will uncomment this once we reach an agreement how to deal with this PR.